.gitignore

```diff
@@ -6,3 +6,4 @@
 
 # direnv (https://direnv.net/)
 .envrc
+.direnv
```

Comment:
Can you say a bit about this file? I tried to find documentation but failed.

Reply:
Looks like the [layout directory](https://github.com/direnv/direnv/blob/6a73cc2d7d13054829501df8756db04e79ff6a29/stdlib.sh#L121) used by direnv. You can see that direnv itself is ignoring this file ([sources](https://github.com/direnv/direnv/blob/master/.gitignore#L2)). People want to add `.direnv` to gitignore, but somehow it always get refused (https://github.com/github/gitignore/pull/4342, https://github.com/github/gitignore/pull/2075).

Reply:
Weird... neither the website nor the man pages mention this. Only understood what this does after looking here https://github.com/direnv/direnv/wiki#project-layouts under Python and Ruby.

It seems there is no Rust layout yet. But ".direnv" is used consistently, so it makes sense to anticipate that 👍🏻

Learned something new about direnv :-)